### DISCUSSION TITLE
How to run set of features or scenarios which are tagged with particular tag e.g.@tag1

### DISCUSSION THREAD

**Discussion by pritesh848** [reqnroll/Reqnroll#347] [Contributor Q&A] [answered]
I am trying to run feature files with tagged as @ro using cmd command
dotnet test MobileAutomation.sln --filter "Traits=RO"

is it possible to run like this in reqnroll ?

getting below error
Starting test execution, please wait...
Logging Vstest Diagnostics in file: C:\MobileAutomationBDD\Mobile-Automation
A total of 1 test files matched the specified pattern.
No test matches the given testcase filter Traits=RO in C:\MobileAutomationBDD\Mobile-Automation\bin\Debug\net8.0\Mobile.dll

**Comment by clrudolphi** [comment, marked as answer]
Have you attempted using --filter "TestCategory=RO" as described on this page of the [documentation?](https://docs.reqnroll.net/latest/execution/executing-specific-scenarios.html)

**Comment by pritesh848** [comment]
Thanks, it was working with TestCategory
